doc/DataPlaneCodeGeneration/DeveloperDrivenEvolution.md

```diff
@@ -4,6 +4,7 @@ Convenience API is an additive layer on top of protocol method so it could produ
 To add a convenience API that takes and returns strongly-typed input and output models instead of raw JSON, steps are:
 1. [Create the models](#create-the-models)
 2. [Create the convenience method with new signature](#create-the-convenience-method-with-new-signature)
+- [Check if convenience method needs](#check-if-convenience-method-needs)
```

Comment:
I'd like to make this step a little more subtle, since in most cases a convenience method will be needed.